src/pages/Admin/Room/Edit/RoomEdit.tsx

```diff
@@ -32,13 +33,16 @@ const EditRoom: React.FC<EditRoomProps> = ({
     watch,
     handleSubmit,
     setValue,
+    errors,
     formState: { isSubmitting, dirty },
   } = useForm({
+    validationSchema: roomEditSchema,
     defaultValues: {
       title: room.title,
       url: room.url,
       description: room.description,
       tempalte: room.template,
```

Comment:
This seems like a bug/typo?
```suggestion
      template: room.template,
```